azurecompute-arm/src/test/java/org/jclouds/azurecompute/arm/features/VirtualNetworkApiLiveTest.java

```diff
@@ -0,0 +1,101 @@
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jclouds.azurecompute.arm.features;
+
+import org.jclouds.azurecompute.arm.domain.VirtualNetwork;
+import org.jclouds.azurecompute.arm.internal.BaseAzureComputeApiLiveTest;
+
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNotNull;
+import static org.testng.Assert.assertTrue;
+
+@Test(groups = "live", singleThreaded = true)
+public class VirtualNetworkApiLiveTest extends BaseAzureComputeApiLiveTest {
+
+    private final String subscriptionid = "subscriptionid";
+    private String resourcegroup;
+
+    @BeforeClass
+    @Override
+    public void setup(){
+        super.setup();
+        resourcegroup = getResourceGroupName();
+    }
+
+    @Test(groups = "live")
+    public void deleteVirtualNetworkResourceDoesNotExist() {
+
+        final VirtualNetworkApi vnApi = api.getVirtualNetworkApi(resourcegroup);
+
+        vnApi.deleteVirtualNetwork(VIRTUAL_NETWORK_NAME);
+
+        //TODO: need to check that we get "no content" back
+
+    }
+
+    @Test(groups = "live", dependsOnMethods = "deleteVirtualNetworkResourceDoesNotExist")
+    public void createVirtualNetwork() {
+
+        final VirtualNetworkApi vnApi = api.getVirtualNetworkApi(resourcegroup);
+
+        //Create properties object
+
+        final VirtualNetwork.VirtualNetworkProperties virtualNetworkProperties =
+                VirtualNetwork.VirtualNetworkProperties.create(null, null,
+                        VirtualNetwork.AddressSpace.create(Arrays.asList(DEFAULT_VIRTUALNETWORK_ADDRESS_PREFIX)), null);
+
+        VirtualNetwork vn = vnApi.createOrUpdateVirtualNetwork(VIRTUAL_NETWORK_NAME, LOCATION, virtualNetworkProperties);
+
+        assertEquals(VIRTUAL_NETWORK_NAME, vn.name());
+        assertEquals(LOCATION, vn.location());
+    }
+
+    @Test(groups = "live", dependsOnMethods = "createVirtualNetwork")
+    public void getVirtualNetwork() {
+
+        final VirtualNetworkApi vnApi = api.getVirtualNetworkApi(resourcegroup);
+        VirtualNetwork vn = vnApi.getVirtualNetwork(VIRTUAL_NETWORK_NAME);
+
+        assertNotNull(vn.name());
+        assertNotNull(vn.location());
+        assertNotNull(vn.properties().addressSpace().addressPrefixes());
+    }
+
+    @Test(groups = "live", dependsOnMethods = "getVirtualNetwork")
```

Comment:
Make this depend on the create method. We want this not to be skipped even if the "get" test fails.